templates/README.md

```diff
@@ -11,10 +11,10 @@ Distributed under the [Boost Software License, Version 1.0](http://www.boost.org
 
 ### Build Status
 
-Branch          | Travis | Appveyor | Coverity Scan | codecov.io | Deps | Docs | Tests |
-:-------------: | ------ | -------- | ------------- | ---------- | ---- | ---- | ----- |
-[`master`](https://github.com/boostorg/template/tree/master) | [![Build Status](https://travis-ci.org/boostorg/template.svg?branch=master)](https://travis-ci.org/boostorg/template) | [![Build status](https://ci.appveyor.com/api/projects/status/github/boostorg/template?branch=master&svg=true)](https://ci.appveyor.com/project/maintainer/template-xyzzy/branch/master) | [![Coverity Scan Build Status](https://scan.coverity.com/projects/BADLE-NUMBER-LIKE-13982/badge.svg)](https://scan.coverity.com/projects/boostorg-template) | [![codecov](https://codecov.io/gh/boostorg/template/branch/master/graph/badge.svg)](https://codecov.io/gh/boostorg/template/branch/master)| [![Deps](https://img.shields.io/badge/deps-master-brightgreen.svg)](https://pdimov.github.io/boostdep-report/master/template.html) | [![Documentation](https://img.shields.io/badge/docs-master-brightgreen.svg)](http://www.boost.org/doc/libs/master/doc/html/template.html) | [![Enter the Matrix](https://img.shields.io/badge/matrix-master-brightgreen.svg)](http://www.boost.org/development/tests/master/developer/template.html)
-[`develop`](https://github.com/boostorg/template/tree/develop) | [![Build Status](https://travis-ci.org/boostorg/template.svg?branch=develop)](https://travis-ci.org/boostorg/template) | [![Build status](https://ci.appveyor.com/api/projects/status/github/boostorg/template?branch=develop&svg=true)](https://ci.appveyor.com/project/maintainer/template-xyzzy/branch/develop) | [![Coverity Scan Build Status](https://scan.coverity.com/projects/BADGE-NUMBER-LIKE-13982/badge.svg)](https://scan.coverity.com/projects/boostorg-template) | [![codecov](https://codecov.io/gh/boostorg/template/branch/develop/graph/badge.svg)](https://codecov.io/gh/boostorg/template/branch/develop) | [![Deps](https://img.shields.io/badge/deps-develop-brightgreen.svg)](https://pdimov.github.io/boostdep-report/develop/template.html) | [![Documentation](https://img.shields.io/badge/docs-develop-brightgreen.svg)](http://www.boost.org/doc/libs/develop/doc/html/template.html) | [![Enter the Matrix](https://img.shields.io/badge/matrix-develop-brightgreen.svg)](http://www.boost.org/development/tests/develop/developer/template.html)
+Branch          | Travis | Appveyor | Azure Pipelines | Coverity Scan | codecov.io | Deps | Docs | Tests |
```

Comment:
Folks will not be using all three, will they?

Reply:
Not necessarily. We use all three for Boost.GIL as the Travis CI is very slow (~24-48h waiting time for build is not uncommon). AppVeyor is Windows only. Then AzP is one that provides feedback on PRs quick. Once PRs are merged, they are ground by the other two.